### PR TITLE
Resolve #1447, #1449, #1384, #1385, #1450, #1451, #1443, #1452, #1453, #1454, #1455, #1456, #1310, #1458, #1459, #1457, #1460

### LINKED ISSUE #1447: TransferAmountTextfield disabled after previous transfer
how to reproduce:
1. Select email transfer
2. Select any account
3. review, and authorize the transfer
4. Click "back to home"
5. Select email transfer
6. Select any account
7. TransferAmountTextfield is disabled due to
accountSelection.status === 'dirty'

we should sync selected account if status === 'dirty' or 'initialized'

### PR DESCRIPTION
**Pull request by timothywangdev**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -241,80 +243,82 @@ class App extends Component {
...
-                <DefaultLayout
-                  path={`${paths.OAuthRedirect}`}
-                  component={userIsAuthenticated(OAuthRedirectComponent)}
-                />
-                <DefaultLayout
-                  path={`${paths.userSetting}`}
-                  component={userIsAuthenticated(UserSettingContainer)}
-                />
-                {process.env.REACT_APP_ENV === 'test' && (
+          <PersistGate loading={null} persistor={persistor}>
```
Comment (InVincible2016) [inline src/App.js:246]: This part of changes does not seem to belong to this issue.

File: src/drive.js [hunk 2]
```diff
@@ -471,6 +465,17 @@ async function saveWallet (walletDataList: any, encryptedWalletFileData: any) {
     name: 'Readme.txt',
     content: ''
   })
+  
+  files = await listFiles(DRIVE_SPACE, null, true, ROOT_FOLDER_NAME)
```
Comment (InVincible2016) [inline src/drive.js:469]: Change issue title to better reflect the problem.

File: src/tests/e2e/directTransferWalletAuth.test.e2e.js [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+                  options: {
+                    variant: 'error'
+                  }
+                }
+              }
+            }
+          ]
+        ),
+        dtAuthPage.connect()
+      ])
```
Comment (InVincible2016) [inline src/tests/e2e/directTransferWalletAuth.test.e2e.js:286]: Missing checking here.

File: src/App.js (hunk 1 above)
Comment (timothywangdev) [reply src/App.js:246]: fixed

File: src/drive.js (hunk 2 above)
Comment (timothywangdev) [reply src/drive.js:469]: fixed

File: src/tests/e2e/directTransferWalletAuth.test.e2e.js (hunk 3 above)
Comment (timothywangdev) [reply src/tests/e2e/directTransferWalletAuth.test.e2e.js:286]: cant wait for btc due to timeout

**Pull request merged**